eclair-core/src/main/scala/fr/acinq/eclair/transactions/CommitmentSpec.scala

```diff
@@ -70,11 +72,17 @@ case class IncomingHtlc(add: UpdateAddHtlc) extends DirectedHtlc
 
 case class OutgoingHtlc(add: UpdateAddHtlc) extends DirectedHtlc
 
-final case class CommitmentSpec(htlcs: Set[DirectedHtlc], feeratePerKw: FeeratePerKw, toLocal: MilliSatoshi, toRemote: MilliSatoshi) {
+final case class CommitmentSpec(htlcs: Set[DirectedHtlc], commitTxFeerate: FeeratePerKw, toLocal: MilliSatoshi, toRemote: MilliSatoshi) {
```

Comment:
My gut feeling is that this class should depend on the `CommitmentFormat` (or `ChannelType` but you get the idea).

Probably okay for now.

Reply:
The reason I didn't go that way is because changing this class means we need to change the channel codecs (and a migration)...and I really wanted to avoid that if that wasn't really necessary!